src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -492,4 +493,24 @@ protected function createChildContext(array $parentContext, $attribute/*, string
 
         return $parentContext;
     }
+
+    protected function attributeAllowedWithVersion(array $context, AttributeMetadataInterface $attributeMetadata): bool
+    {
+        $sinceVersion = $attributeMetadata->getSince();
+        $untilVersion = $attributeMetadata->getUntil();
+
+        if (!isset($context['version'])) {
```

Comment:
Now you should check the default context too.

Reply:
Please, can you tell me more about the default context ?